portfolio/src/main/webapp/index.html

```diff
@@ -3,106 +3,172 @@
+            </a>
           </li>
-          <li>
-            <h3>Pomoro</h3>
+          <li class="project-card">
+            <h3>Pomoro</a></h3>
             <h4>Personal Project</h4>
             <p>
               Online timer for using the pomodoro technique. It focuses on 
-              simplicity and usability. Tech Stack: HTML, CSS, JavaScript, Vue.js
+              simplicity and usability. 
             </p>
+            <p>
+              Tech Stack: HTML, CSS, JavaScript, Vue.js
+            </p>
+            <a href="https://github.com/malcolmrjones/pomoro">Learn More</a>
           </li>
-          <li>
+          <li class="project-card">
             <h3>BBConnection</h3>
             <h4>Hackathon - HackNCAT 2018</h4>
             <p>
               Web platform that connects small and medium black-owned 
-              businesses with opportunities. Tech Stack: HTML, CSS, JavaScript
+              businesses with opportunities. 
+            </p>
+            <p>
+              Tech Stack: HTML, CSS, JavaScript
             </p>
           </li>
-          <li>
+          <li class="project-card">
             <h3>Rolisto</h3>
             <h4>Personal Project</h4>
             <p>
-              Web app to create and manage task and to-do lists. 
+              Web app to create and manage task lists and to-do lists. 
+            </p>
+            <p>
               Tech Stack: HTML, CSS, JavaScript, Angular, Firebase Firestore
             </p>
           </li>
-          <li>
+          <li class="project-card">
             <h3>Roobeks Timer</h3>
             <h4>Personal Project</h4>
             <p>
               Easy to use timer and tool for Rubik's Cube solves. 
+            </p>
+            <p>
               Tech Stack: HTML, CSS, JavaScript
             </p>
+            <a href="https://github.com/malcolmrjones/RoobeksTimer">Learn More</a>
           </li>
         </ul>
-
-        <hr>
-
-
-        <p>Click the button below to receive a warm greeting!</p>
-        <button onclick="addGreeting()">Greet Me</button>
-        <div id="greeting-container"></div>
-
-        <hr>
-
-        <h2>Comments</h2>
-
-        <form action="/data" method="post">
-          <label for="comment-author">Name: </label>
-          <input type="text" name="comment-author"/>
-          <input type="text" name="comment-input"/>
-          <input type="submit" value="COMMENT"/>
-        </form>
-
-        <button onclick="fetchComments()">Load Comments</button>
-        <label for="comment-count">Number of comments to display: </label>
-        <input type="number" id="comment-count" name="comment-count" min="1" max="10">
-        <ul id="commentlist"></ul>
-
-        <button onclick="deleteAllComments()">Delete All Comments</button>
+      </div>
+      
+      <div class="content-card">
+        <h2 id="gallery-section">Gallery</h2>
+        <img src="images/gallery/pic-1.jpg" class="gallery-img" id="image-view" />
```

Comment:
Nit: view usually has a fullscreen connotation so I would avoid using that word.